docs/src/integrate/patina_dxe_core_requirements.md

```diff
@@ -280,10 +265,91 @@ and MemoryAttributesProtocol->SetMemoryAttributes() calls that pass in EFI_MEMOR
 memory (typically representing device memory) should be marked as executable. For AArch64, the ARM ARM v8 B2.7.2 states
 that having executable device memory (what EFI_MEMORY_UC maps to) is a programming error and has been observed to cause
 crashes due to speculative execution trying instruction fetches from memory that is not ready to be touched or should
-never be touched. For x86 platforms, we still should protect these ranges to prevent devices having access to executable
+never be touched. For x86 platforms, these regions are still protected to prevent devices having access to executable
 memory.
 
-### 4. Known Limitations
+### 4. Architectural Requirements
+
+This section details Patina requirements that are specific to a particular CPU architectural requirements.
+
+#### 4.1 CpuDxe Is No Longer Used
+
+EDK II supplies a driver named `CpuDxe` that provides CPU related functionality to a platform. In Patina DXE Core, this
+is part of the core, not offloaded to a driver. As a result, the CPU Arch and memory attributes protocols are owned by
+the Patina DXE Core. MultiProcessor (MP) Services are not part of the core (see following sections for specific
+guidance on MP Service enabling for specific architectures).
+
+> **Guidance:**
+> Platforms must not include `CpuDxe` in their platforms and instead use CPU services from Patina DXE Core and MP
+> Services from a separate C based driver.
+
+#### 4.2 AArch64-specific requirements
+
+This section details Patina requirements specific to the AArch64 architecture.
+
+##### 4.2.1 Aarch64 Generic Interrupt Controller (GIC)
+
+On AArch64 systems, when Patina assumes ownership of `CpuDxe` it also encompasses the functionality provided by the
+`ArmGicDxe` driver which configures the Aarch64 GIC. This is because GIC support is a prerequisite for `CpuDxe` to
+handle interrupts correctly. As such, AArch64 platforms also should not include the `ArmGicDxe` driver.
+
+In addition, different AArch64 platforms can place the GIC register set at different locations in the system memory map.
+Consequently, AArch64 platforms must supply the base addresses associated with the GIC as part of the `CpuInfo` struct
+that is part of core configuration.
+
+> **Guidance:**
+> AArch64 platforms should not use the `ArmGicDxe` driver and must supply the base addresses of the GIC in the `CpuInfo`
+> structure.
+
```

Comment:
```suggestion
> Platforms must not include `CpuDxe` in their platforms and instead use CPU services from Patina DXE Core.
```

Just for clarity, the CpuDxe should be mentioned here as well to follow the x64 guidance.

Reply:
I misread that then, if its under general, then it doens't need to be added under x64/AArch64.